raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -189,7 +189,7 @@ def filter_reachable_routes(
 
     for route in routes:
         node_network_state = nodeaddresses_to_networkstates.get(
-            route.node_address, NODE_NETWORK_UNREACHABLE
+            route.route[1], NODE_NETWORK_UNREACHABLE
```

Comment:
May i suggest introducing a `next_hop_address` method on the `RouteState` where `route[1]` is used?